src/Panel/TimeLayerSliderPanel/TimeLayerSliderPanel.tsx

```diff
@@ -0,0 +1,496 @@
+import * as React from 'react';
+
+import moment from 'moment';
+
+const _isFinite = require('lodash/isFinite');
+const _isEqual = require('lodash/isEqual');
+
+import {
+  TimeSlider,
+  timeLayerAware,
+  ToggleButton,
+  SimpleButton
+} from '../../index';
+
+import {
+  Select,
+  DatePicker,
+  Popover
+} from 'antd';
+const RangePicker = DatePicker.RangePicker;
+const Option = Select.Option;
+
+import './TimeLayerSliderPanel.less';
+
+type timeRange = [moment.Moment, moment.Moment];
+
+export interface Tooltips {
+  hours: string;
+  days: string;
+  weeks: string;
+  months: string;
+  years: string;
+  setToNow: string;
+  dataRange: string;
+}
+
+export interface DefaultTimeLayerSliderPanelProps {
+  className: string;
+  onChange: (arg: moment.Moment) => void;
+  timeAwareLayers: any[];
+  value: moment.Moment;
+  dateFormat: string;
+  tooltips: Tooltips;
+}
+
+export interface TimeLayerSliderPanelProps extends Partial<DefaultTimeLayerSliderPanelProps> {
+  map: any;
+  initStartDate: moment.Moment;
+  initEndDate: moment.Moment;
+}
+
+export interface TimeLayerSliderPanelState {
+  value: moment.Moment;
+  playbackSpeed: string;
+  autoPlayActive: boolean;
+  startDate: moment.Moment;
+  endDate: moment.Moment;
+}
+
+/**
+ * mapStateToProps - mapping state to props of Map Component.
+ *
+ * @param {Object} state current state
+ * @return {Object} mapped props
+ */
```

Comment:
Please remove unneeded multiline comment.

Reply:
… also remove `{Object}`. The types are determined by TypeScript now.